geopandas/explore.py

```diff
@@ -367,7 +367,11 @@ def my_colormap(value):  # scalar value defined in 'column'
                     "Cannot specify 'categories' when column has categorical dtype"
                 )
             categorical = True
-        elif gdf[column].dtype is np.dtype("O") or categories:
+        elif (
+            gdf[column].dtype is np.dtype("O")
+            or gdf[column].dtype is np.dtype(bool)
```

Comment:
Thinking about more edge cases should we also catch the string dtype case? `gdf[column].dtype=="string"` and `gdf[column].dtype.type is str`? I suppose date time would be similar, but less likely to come up.

Reply:
In general better supporting pandas extension dtypes such as `string` might be nice (but maybe better for a separate PR), as for example also for boolean data you now have `boolean` dtype (and specifically for that, using a helper like `pd.api.types.is_bool_dtype` could be useful, as it will catch both the numpy bool dtype and extension boolean dtype)

Reply:
I'll open an issue for that and leave it for another PR. The same should happen in `plot`.